sheetScript.gs

```diff
@@ -75,9 +75,15 @@ function updatePlaylists(sheet) {
   /// For each playlist...
   for (var iRow = reservedTableRows; iRow < sheet.getLastRow(); iRow++) {
     Logger.clear();
-    Logger.log("Row: " + (iRow+1));
+    Logger.log("Row: " + (iRow + 1));
     var playlistId = data[iRow][reservedColumnPlaylist];
-    if (!playlistId) continue;
+    if (!playlistId || playlistId.substring(0, 1) == "#")
```

Comment:
oh ya I also added commenting out lines for playlists starting with `#` and would really love to keep this but it's a bit out of scope